### DISCUSSION TITLE
Weather

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#623] [General]
How do you like the weather guys?

**Comment by MirkosMangle** [comment]
That's something not for me, i feel extremely hot and can't do anything outside.

**Comment by hernessteve** [comment]
It is extremely hot in my area too. The only thing i manage to do these days - sit under my air conditioner and read my papers on "black lives matter" [https://writinguniverse.com/free-essay-examples/blаck-lives-matter/](https://writinguniverse.com/free-essay-examples/black-lives-matter/). We have a competition at work for the best essay and looks like the prize gonna be mine. Same like the previous year.

**Comment by k-mizoguch1** [comment]
plz https://github.com/kzu000/Food-what-do-you-like
